src/wallet/rpc/wallet.cpp

```diff
@@ -610,6 +610,27 @@ RPCHelpMan simulaterawtransaction()
             RPCResult::Type::OBJ, "", "",
             {
                 {RPCResult::Type::STR_AMOUNT, "balance_change", "The wallet balance change (negative means decrease)."},
+                {RPCResult::Type::ARR, "details", "",
+                {
+                    {RPCResult::Type::OBJ, "", "",
+                        {
+                            {RPCResult::Type::STR_HEX, "txid", "the transaction hex of transaction provided by the user"},
+                            {RPCResult::Type::ARR, "wallet_conflicts", "list of txids of transactions in the wallet that conflict with this transaction",
+                            {
+                                {RPCResult::Type::STR_HEX, "txid_in_wallet", "the txid of conflicting transactions in wallet"},
```

Comment:
In bf9deb407a8d73ac3b645b34c9bd6d28729a809c "rpc/wallet: Add details and duplicate section for simulaterawtransaction"

`txid` is sufficient

```suggestion
                                {RPCResult::Type::STR_HEX, "txid", "the txid of conflicting transactions in wallet"},
```